Add blood vessel dataset processing script (#184)
open-mmlab/mmsegmentation: docs/getting_started.md

python tools/convert_datasets/pascal_context.py data/VOCdevkit data/VOCdevkit/VOC2010/trainval_merged.json
```

### CHASE DB1

The training and validation set of CHASE DB1 could be download from [here](https://staffnet.kingston.ac.uk/~ku15565/CHASE_DB1/assets/CHASEDB1.zip).

> To convert CHASE DB1 dataset to mmsegmentation format, you should run the following command:

Reviewer: ```suggestion
To convert CHASE DB1 dataset to MMsegmentation format, you should run the following command:
```

```shell
python tools/convert_datasets/chase_db1.py ~/CHASEDB1.zip
```

The script will make directory structure automatically.

### DRIVE

> The training and validation set of DRIVE could be download from [here](https://drive.grand-challenge.org/). Before that, you should registrate an account.

Reviewer 2: Now, there is no '1st_manual' in the test dataset.

> The training and validation set of DRIVE could be download from [here](https://drive.grand-challenge.org/). Before that, you should registrate an account. Now, there is no '1st_manual' in the test dataset, if you need it, please search for it by yourself, we do not provide links to download.

Reviewer: ```suggestion
The training and validation set of DRIVE could be download from [here](https://drive.grand-challenge.org/). Before that, you should register an account. Currently '1st_manual' is not provided officially. 
```

To convert DRIVE dataset to mmsegmentation format, you should run the following command:

```shell
python tools/convert_datasets/drive.py ~/training.zip ~/test.zip
```

The script will make directory structure automatically.

### HRF

First, download [healthy.zip](https://www5.cs.fau.de/fileadmin/research/datasets/fundus-images/healthy.zip), [glaucoma.zip](https://www5.cs.fau.de/fileadmin/research/datasets/fundus-images/glaucoma.zip), [diabetic_retinopathy.zip](https://www5.cs.fau.de/fileadmin/research/datasets/fundus-images/diabetic_retinopathy.zip), [healthy_manualsegm.zip](https://www5.cs.fau.de/fileadmin/research/datasets/fundus-images/healthy_manualsegm.zip), [glaucoma_manualsegm.zip](https://www5.cs.fau.de/fileadmin/research/datasets/fundus-images/glaucoma_manualsegm.zip) and [diabetic_retinopathy_manualsegm.zip](https://www5.cs.fau.de/fileadmin/research/datasets/fundus-images/diabetic_retinopathy_manualsegm.zip).

To convert HRF dataset to mmsegmentation format, you should run the following command:

```shell

> python tools/convert_datasets/hrf.py ~/healthy.zip ~/healthy_manualsegm.zip ~/glaucoma.zip ~/glaucoma_manualsegm.zip ~/diabetic_retinopathy.zip ~/diabetic_retinopathy_manualsegm.zip

Reviewer: We may omit `~`. We may use `/path/to/healthy.zip` instead.

Author: OK

```

The script will make directory structure automatically.

### STARE

First, download [stare-images.tar](http://cecas.clemson.edu/~ahoover/stare/probing/stare-images.tar), [labels-ah.tar](http://cecas.clemson.edu/~ahoover/stare/probing/labels-ah.tar) and [labels-vk.tar](http://cecas.clemson.edu/~ahoover/stare/probing/labels-vk.tar).

To convert STARE dataset to mmsegmentation format, you should run the following command:

```shell

> python tools/convert_datasets/hrf.py ~/stare-images.tar ~/labels-ah.tar ~/labels-vk.tar

Reviewer: Should be `stair.py`?

Author: Ah......yes, I will fix it.
